build/common.props

```diff
@@ -20,6 +20,7 @@
     <PackageLicense>https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/master/LICENSE.txt</PackageLicense>
     <SignAssembly>true</SignAssembly>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsNotAsErrors>0618</WarningsNotAsErrors>
```

Comment:
this is OK in test, but for src use individual.

Reply:
Makes sense!